ENV.MD

```diff
@@ -0,0 +1,427 @@
+# Environment variables
+
+Please check [how the container starts](https://github.com/tomav/docker-mailserver/blob/master/target/start-mailserver.sh) to understand what's expected. Also if an option doesn't work as documented here, check if you are running the latest image!
```

Comment:
Instead of pointing towards the start-mailserver.sh script to "understand whats expected" the script should use default values for all configuration options.

Reply:
I am actually not sure what to expect from so many lines of code.

Reply:
Indeed. its good to point to the script for "deeper understanding", but the container should start without any of the explained values explicitly set. (or them being set through compose).